.github/workflows/deploy-apis-to-production.yml

```diff
@@ -424,29 +424,29 @@ jobs:
         uses: docker/build-push-action@v4.1.1
         with:
           push: true
-          context: src/airflow
+          context: src/workflows
           target: deployment
-          tags: ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-apache-airflow:${{ needs.image-tag.outputs.build_id }},${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-apache-airflow:latest
```

Comment:
NB: Could cause some issues with deployment.

Reply:
Thanks @Mnoble-19 for the clarifications. Appears we might need to have bi-weekly DevOps workshops where we can collaboratively work on these things. ADDITIONALLY, we might want to work on them and test them over night times or weekends as a safety measure. cc: @BenjaminSsempala